lua/nvim-treesitter/highlight.lua

```diff
@@ -81,14 +81,20 @@ function M.attach(bufnr, lang)
   local parser = parsers.get_parser(bufnr, lang)
   local config = configs.get_module('highlight')
 
-  for k, v in pairs(config.custom_captures) do
-    hlmap[k] = v
+  if config.use_languagetree then
+    print("Warning, you are using the languagetree, a very unstable feature")
```

Comment:
not sure this is useful

Reply:
I wonder too now that I added the documentation for it.